test.py

```diff
@@ -18,49 +18,50 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 # MA 02110-1301, USA
 
+from __future__ import absolute_import
+from __future__ import print_function
+
 import cabarchive as cab
 import datetime
 import subprocess
 import time
 import hashlib
 
-def check_archive(filename):
+
+def check_archive(filename, expected_rc=0):
     argv = ['cabextract', '--test', '/tmp/test.cab']
     rc = subprocess.call(argv)
-    assert rc == 0
+    assert rc == expected_rc, "inavlid return code: %r" % rc
```

Comment:
invalid

Reply:
Fixed in dc9a2c2f7f531ea3051d7a37f8fc91ff95f21655